### PR TITLE
Refactor to use javascript classes

### PR DESCRIPTION
**Pull request by berylcl**
** In this project i was able to**
:white_check_mark:Use JavaScript classes.
:white_check_mark:use medium-fidelity wireframes to create a UI.

### CODE REVIEW

**Review by Shisui6 — COMMENTED** [review]
Hi @berylcl, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🛠️

You have submitted a pair programming project with no commits from your coding partner. Please make sure that your coding partners commit to this project and submit again for review

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!🍻💻

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid, unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: style.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+#add-book-form button {
+  font-size: 16px;
+  padding: 5px 15px;
+  background-color: #7b2ea8;
+  color: #fff;
+  border: none;
+  border-radius: 3px;
+}
+
```
Comment (rloterh) [inline style.css]: Your app is working as intended 👌, however, in this activity, we want the UI design of this book app to closely match the given [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/bb245ca8a9d3bb04ebefbbd45e800764e1fc5f74/books/images/awesome_books_core_elements.png). Let's, therefore, introduce the following improvements to our design:
- [ ] The _Add New Book_ form should be vertically center aligned with the _Books table_.

- [ ] The _Add_ button on the _Add New Books_ form should be aligned to the right with the input fields, as seen in the wireframe.

**Your design:**
image

**Wireframe design:**
image

File: README.md [hunk 2]
```diff
@@ -1,29 +1,99 @@
...
+### Run tests
+
+To run tests, run the following command:
+
+### Deployment
+
+You can deploy this project using:git pages
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
```
Comment (rloterh) [inline README.md]: - [ ] [OPTIONAL]
Kindly include clear and step-by-step instructions in your ReadMe on how to _setup_ and _run_ your app, under the _Getting Started_ section. 
Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Kingsley & Gakii, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 
🚦 _ 🏡 🏃‍♂️ 🏚️ 🌴 🗼

## Highlights 👍  
- Objects properly refactored into classes!
- App is running correctly after refactoring!

## Required Changes ♻️
_Kindly check the inline comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

_Kindly check the inline comments marked **[OPTIONAL]** under the review._

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
Hi @berylcl and @unorjikingsley,

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

# Status: Approved ✔️ 

Cheers and Happy coding!👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**